edi_voxel_sale_order_import/tests/test_voxel_sale_order_import.py

```diff
@@ -45,6 +45,18 @@ def setUpClass(cls):
         cls.product_test_2 = cls.env["product.product"].create(
             {"default_code": "222222", "name": "PRODUCT TEST 2"}
         )
+        # Hypothetical unit of measure to be able to load the test file
+        # catching the voxel unit of measure called 'Cajas' for the
+        # first order line
+        cls.boxes_uom = cls.env["uom.uom"].create(
+            {
+                "name": "Boxes 3x3x3",
+                "voxel_code": "Cajas",
+                "category_id": cls.env.ref("uom.product_uom_unit").id,
+                "uom_type": "bigger",
+                "factor_inv": 9.0,
+            }
+        )
 
     def test_create_document_from_xml(self):
```

Comment:
You should isolate this method into another class inheriting from the main one for avoiding to be executed twice, once here, and another on the new module.